### ISSUE TITLE
fetchEdgeProps method has an error in fetch.ts

### ISSUE THREAD

**Issue by devenami** [vesoft-inc/nebula-studio#72]
```code
export async function fetchEdgeProps(payload: {
  idRoutes: string[];
  type: string;
  edgeFields?: any;
}) {
  const { idRoutes, edgeFields, type } = payload;
  const edgeType = '`' + type + '`';
  let gql = `fetch prop on ${edgeType} ${idRoutes.join(', ')}`;
  if (edgeFields) {
    gql += ` yield ${edgeType}._src, ${edgeType}._dst `;
    edgeFields[type].forEach(edgeField => {
      if (edgeField !== 'type') {
        gql += `,${edgeType}.${edgeField}`;
      }
    });
  } else {
    gql += ' YIELD edge as `_edge`';
  }
```
`_edge`must not start with _

**Comment by hetao92** [comment]
Thx. We will fix this. Btw, master branch is our development branch. This is used for nebula nightly version with http-gateway nightly version. If you connect nebula v2.6.x with gateway v2.1.x, please use studio v3.1.0 better

**Comment by hetao92** [comment]
fixed in #74
